### PR TITLE
TECH-1891 Story Title Bugfix

### PR DESCRIPTION
**Pull request by sw3dish**
** Which issue(s) does this PR address? **

[TECH-1891: Story_title is the same for every smt story in the scraped data](https://industrydive.atlassian.net/browse/TECH-1891)

** What does this PR do? **

This PR fixes the issue of the story_title being set to the same value for every SMT story in the scraped data. Instead of setting story_title to the text in the text in the first "dc:title"
attribute, usually one of the links in the header menu, story_title is now assigned the value of the text in the first "dc:title" attribute in the ** section#section-content **. This finds the title that is displayed in the article.

It also adds a convenience testing script -- test_crawl_smt.py

** Where should the reviewer start? **

Run the included `test_crawl_smt.py` file. Notice that the number of unique story_titles = the number of unique titles

** How should this be manually tested? **

- Download only the `test_crawl_smt.py` file (or make the changes manually). 
- Execute `python test_crawl_smt.py --limit 10`
- Notice that the number of titles differs from the number of story titles
- Checkout this branch
- Execute `python test_crawl_smt.py --limit 10`
- Notice that the number of titles is the same as the number of story titles

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,6 @@
 *.html
 urllog.txt
 *log
+.DS_Store
+tox.ini
```
Comment (dbarbar) [inline .gitignore]: Why tox.ini? Generally it defines the rules for running tests and static code analysis. I don't think it should be in this list.

File: scraper/utils.py [hunk 2]
```diff
@@ -171,6 +171,7 @@ def get_author_social_urls(page):
     social_network_link_ids = [
         'facebook',
         'twitter',
+        'linkedin',
```
Comment (dbarbar) [inline scraper/utils.py:174]: Were we not getting linkedin data before?

File: .gitignore (hunk 1 above)
Comment (sw3dish) [reply .gitignore]: Whoops... it wasn't in there before, and I had just added it so my linter would follow the CircleCI settings

File: scraper/utils.py (hunk 2 above)
Comment (sw3dish) [reply scraper/utils.py:174]: It was removed at some point

**Pull request merged**